clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -5,13 +5,16 @@ import { ConnectionClosed, DataReceived } from "./Common";
 import { DefaultHttpClient, HttpClient } from "./HttpClient";
 import { IConnection } from "./IConnection";
 import { ILogger, LogLevel } from "./ILogger";
+import { HttpTransportType, ITransport, TransferFormat } from "./ITransport";
 import { LoggerFactory } from "./Loggers";
-import { ITransport, LongPollingTransport, ServerSentEventsTransport, TransferFormat, TransportType, WebSocketTransport } from "./Transports";
+import { LongPollingTransport } from "./LongPollingTransport";
+import { ServerSentEventsTransport } from "./ServerSentEventsTransport";
 import { Arg } from "./Utils";
+import { WebSocketTransport } from "./WebSocketTransport";
 
 export interface IHttpConnectionOptions {
     httpClient?: HttpClient;
-    transport?: TransportType | ITransport;
+    transport?: HttpTransportType | ITransport;
```

Comment:
Does TS support flags? Is this a single transport type or multiple? I'm just wondering if this should be puralized like in C# client?

Reply:
This wasn't configured as a flags for some reason. I'm not sure if @moozzyk tried it and it didn't work or just didn't do it. I'll file an issue and work it out today.

Reply:
https://github.com/aspnet/SignalR/issues/1961

Reply:
There was no fallback originally so flags did not make much sense.